test/exception_summary.jl

```diff
@@ -0,0 +1,142 @@
+using Test
+using ExceptionUnwrapping: summarize_current_exceptions, TITLE, INDENT_LENGTH, SEPARATOR
+
+function get_current_exception_string()
+    io = IOBuffer()
+    summarize_current_exceptions(io)
+    str = String(take!(io))
```

Comment:
Up to you, but this is shorthand for the same thing you have here:
```suggestion
    sprint(summarize_current_exceptions)
```

Reply:
Oh cool, I learned something new 😁 I added `str = ` your suggestion, and accepted it.